Example/ReadabilityKit/DetailsController.swift

```diff
@@ -30,12 +30,14 @@ class DetailsController: UIViewController {
     @IBOutlet weak var imageView: UIImageView?
     @IBOutlet weak var videoURLTextView: UITextView?
     @IBOutlet weak var contentTextView: UITextView?
+    @IBOutlet weak var dateText: UILabel!
     
-	var image: UIImage?
+    var image: UIImage?
 	var titleText: String?
 	var desc: String?
 	var keywords: [String]?
 	var videoURL: String?
+    var dateVlaue : String?
```

Comment:
Need to correct typo: `dateVlaue` to `dateValue`

Reply:
corrected in new commit